cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -261,7 +261,7 @@ func StartControllers(s *options.CMServer, kubeClient *client.Client, kubeconfig
 				metrics.DefaultHeapsterService,
 				metrics.DefaultHeapsterPort,
 			)
-			podautoscaler.NewHorizontalController(hpaClient.Core(), hpaClient.Extensions(), hpaClient, metricsClient).
+			podautoscaler.NewHorizontalController(hpaClient.Core(), hpaClient.Autoscaling(), hpaClient, metricsClient).
```

Comment:
This is problematic. We decided not to do this as then a new controller manager won't work with an old apiserver.